package.json

```diff
@@ -17,7 +17,7 @@
     "cover:check": "npm run cover && nyc check-coverage --lines 95 --functions 95 --branches 95",
     "jsdoc": "jsdoc ./src/scripts -r -d _dev/jsdoc && gulp serve --root=./_dev/jsdoc --dev",
     "gulpdoc": "jsdoc gulps/ gulpfile.js -r -d _dev/gulpdoc && gulp serve --root=./_dev/gulpdoc --dev",
-    "eslint": "eslint -c .eslintrc.js --fix --ignore-pattern './src/scripts/modules' ./src/scripts/",
+    "eslint": "eslint -c .eslintrc.js --fix ./src/scripts/",
```

Comment:
It is ignored on my machine (Arch Linux) :thinking:

Reply:
What do you mean by "it's ignored" ?